packages/nimble-components/src/select/testing/select.pageobject.ts

```diff
@@ -68,7 +83,11 @@ export class SelectPageObject {
             throw new Error('Select must be open to click selectedItem');
         }
 
-        this.clickOption(this.selectElement.selectedIndex);
+        const selectedOption = this.getActiveOption();
```

Comment:
Should this function be renamed to `clickActiveItem`? You aren't clicking the selected item anymore.

Reply:
Ehhh...maybe? I hesitate to do that at the moment, as this would technically be a breaking change (though I see no usages of this method outside of our repo). I'll leave open for others to comment on.

Reply:
Don't fear breaking changes, especially when they don't impact any existing clients! Nimble components major version is only at 28 so we still have plenty more versions to bump to before we run out of numbers.